kubernetes/ingress-production.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: mad-learning-web
+  namespace: mad-learning
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+    kubernetes.io/ingress.class: nginx
+spec:
+  rules:
+    - host: learning.mad.itera.no
+      http:
+        paths:
+          - backend:
+              serviceName: mad-learning-web
+              servicePort: 8080
+            path: /
+    tls:
+        - hosts:
+            - learning.mad.itera.no
+        secretName: tls-secret
```

Comment:
An observation related to MAD Platform here:
Currently we wanted ingress to be created by the individual projects, but that can result in collisions for both domain names and secret names (cert-manager generates the secrets). I think maybe ingress should be defined in mad-platform. Thoughts @oyvindne ?

Reply:
@anhtin you can update the secretName to `mad-learning-web-tls-secret` and add it to the pipeline.

Reply:
Yap, will do!